benefits/core/admin.py

```diff
@@ -151,3 +152,11 @@ def add_staff_user_to_group(user, request):
     if user.email in settings.GOOGLE_SSO_STAFF_LIST:
         staff_group = Group.objects.get(name=settings.STAFF_GROUP_NAME)
         user.groups.add(staff_group)
+
+
+def add_transit_agency_staff_user_to_group(user, request):
+    user_sso_domain = user.email.split("@")[1]
+    if user_sso_domain:
+        agency = models.TransitAgency.objects.filter(sso_domain=user_sso_domain).first()
+        if agency is not None and agency.staff_group:
+            user.groups.add(agency.staff_group)
```

Comment:
This feels backwards? Why don't we add the user to the staff group? I guess it results in the same thing, but just looks a little strange.

Reply:
Yeah, it can be done either way. I can change it

Reply:
It is a little weird code-wise. But it also kinda makes sense visually in the Admin tool too, because in order to add a User into a Group in the Admin, I go find the user (http://localhost:11369/admin/auth/user/3/change/) and then add or remove the User's group from this list:
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/1e0685f9-956f-4b8b-bab5-6f00bd13da88">

Reply:
Groups view - Pretty much only has Group name and permissions. You can't even see who is in the group from this view by default
<img width="1500" alt="image" src="https://github.com/user-attachments/assets/f095819a-9bb6-4438-9812-84e7f6fb0aa4">

Reply:
Thanks for those screenshots @machikoyasuda. Either way, whatever makes the most sense to everyone.

Reply:
I went ahead and changed it in https://github.com/cal-itp/benefits/pull/2325/commits/04d3c194d80ae4be4afffd92fefeef986410e93a which I think does match the function name better.

And thanks for those screenshots @machikoyasuda to note how the admin interface presents it.